caddy/README.md

```diff
@@ -29,55 +29,112 @@ Here's the things we find most useful:
 caddy file-server --browse --listen :4040
 ```
 
+### How to serve HTTPS on localhost
+
+Caddy can be used to test with https on localhost.
+
+`Caddyfile`:
+
+```Caddyfile
+https://localhost:443 {
```

Comment:
```suggestion
localhost {
```

Reply:
I've had issues with this. I believe it had something to do with browser caching / redirects.

I've found only running localhost on 443 to be more reliable than running on 80 and 443.

Reply:
These two are exactly the same though. Specifying `https://` still does redirects from HTTP, and serves on port 443 by default, just as the hostname alone does.

Reply:
Noted. I'll switch it back.